ci/images/go-110/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM buildpack-deps:stretch-scm
+
+# asdf
+RUN curl -L https://github.com/asdf-vm/asdf/archive/v0.5.0.tar.gz > asdf.tar.gz && \
+    tar -xzvf asdf.tar.gz && \
+    mv asdf-0.5.0 /asdf
+
+ENV PATH="/asdf/shims:/asdf/bin:${PATH}"
+
+# gcc for cgo
```

Comment:
Much of this seems to be copied from the golang Dockerfile: https://github.com/docker-library/golang/blob/fa0223eaa427188e0c70025f557d515129a9973f/1.10/stretch/Dockerfile

Rather than duplicating it here, consider using the library/golang as your base image with FROM and then layer the asdf-vm stuff on top? It would make this file more concise and add the flexibility of changing to a newer version of the golang base image by simply changing the FROM's URI.

Reply:
Will do, for some reason I had read `rm -rf /var/lib/apt/lists/*` as `rm -rf  /etc/apt/sources.list` and thought they were removing all the configured package sources so copied all this over to prevent that.